### ISSUE TITLE
Design UI/UX

### ISSUE THREAD

**Issue by 0x736E** [Tog-Hackerspace/Website#5]
Options:
- Design from scratch
- Find suitable template to customise

**Comment by edwardgalligan** [comment]
This is super subjective, so hopefully it doesn't get bikesheddy, but I've always really liked the current design.

What would people think of just doing a clean and simple minor update/modernisation/responsivisation of it?

**Comment by edwardgalligan** [comment]
Something like https://rawgit.com/lucideer/eea902a8d480f0f09ec49cdf0da50ce7/raw/498038e3f37f2abf825076242b35e89535166ded/tog.html

**Comment by 0x736E** [comment]
I would agree. The clean non-opinionated style is a good pick for Tog. It's main failure is proper accessibility, mobile support, and inconsistencies overall.

For our benefit, keeping to such a non-opinionated theme should require the least engineering effort to achieve, so we have that going for us also.

**Comment by edwardgalligan** [comment]
I'd be happy to develop out on the above-linked theme if there aren't major objections to the idea from people (it's mobile friendly sofar, as little of it as there is)
